packages/manager/modules/anthos/src/dashboard/ips/translations/Messages_fr_FR.json

```
@@ -0,0 +1,22 @@
+{
+  "anthos_dashboard_ips_title": "Addresses IP",
+  "anthos_dashboard_ips_description": "Dans la console Anthos, vous avez besoin d’allouer des IPs à chacun de vos déploiements Anthos. Vous pouvez assigner autant d’IPs/ranges d’IPs que vous voudrez à votre déploiement Anthos. Vous bénéficiez également de 50 IPs publiques et pouvez en commander de nouvelles pour un faible tarif fixe.",
+  "anthos_dashboard_ips_datagrid_public_actions_menu": "Actions",
+  "anthos_dashboard_ips_datagrid_public_actions_order_ip": "Commander des IP publiques",
```

```suggestion
  "anthos_dashboard_ips_datagrid_public_actions_order_ip": "Commander des IPs publiques",
```
sometimes we pluralize, sometimes we don't. I'm not sure what the rule is but we should be consistent across all content

---

Same as here:
https://github.com/ovh/manager/pull/5456#discussion_r699095040